composer/dag_test_utils/README.md

```diff
@@ -0,0 +1,30 @@
+# Cloud Composer DAG Testing Utility
+
+This package is used internally to unit test the validity of all Cloud Composer sample DAGs. It is not supported for external production use.
+
+## Instructions
+
+Add the following to your `requirements-test.txt` file:
+
+`git+https://github.com/GoogleCloudPlatform/python-docs-samples.git#egg=dag_test_utils&subdirectory=composer/dag_test_utils`
+
+Import the unit testing module
+
+```python
+import unit_testing
```

Comment:
I know this isn't for external use, but do we want to rename this to something composer/airflow-specific?

Reply:
`dag_test_utils` wouldn't be a bad name.

Reply:
+1 to a more specific name

Reply:
After talking to Tim, changed it to `internal_unit_testing` to discourage any production usage